modular_robot_physical/revolve2/modular_robot_physical/_physical_robot_config.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import annotations
+
+import pickle
+from dataclasses import dataclass
+
+from revolve2.modular_robot import ModularRobot
+from revolve2.modular_robot.body.base import ActiveHinge
+
+
+@dataclass
+class PhysicalRobotConfig:
+    """The configuration for running a physical robot."""
+
+    modular_robot: ModularRobot  # the modular robot object.
+    hinge_mapping: dict[ActiveHinge, int]  # which servo is triggered by which hinge.
+    simulation_time: int  # in seconds.
+    control_frequency: int
+    inverse_servos: bool
```

Comment:
# Here you can implement your controller.
            # The current controller does nothing except for always settings the joint positions to 0.5.

Reply:
i see ill add that👍🏻